docs/tutorial.md

```diff
@@ -54,19 +53,19 @@ spec:
     name: echo-hello-world
 ```
 
-To apply the yaml files, use the following command:
+Apply your YAML files to the build as follows:
 
 ```bash
 kubectl apply -f <name-of-file.yaml>
 ```
 
-To see the output of the `TaskRun`, use the following command:
+To check whether running your `Task` succeeded, use the following command:
```

Comment:
```suggestion
To check whether running your `TaskRun` succeeded, use the following command:
```

Reply:
Nit, but doesnt' a TaskRun run your Task?

Reply:
Ah, I see the running part that makes this confusing. Maybe it would be better to say `To check whether your TaskRun succeeded, use the following command:`? TaskRuns are what have the status of success or failure and the command used below is for a TaskRun (i.e. tkn taskrun describe`).

Reply:
That makes perfect sense, I'll chage it to `TaskRun`.